docs/src/dataDisplay/table/index.jsx

```diff
@@ -1,210 +1,443 @@
+                        A standard table.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSample').default}
+                >
+                    <TableSample />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Celled Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s cells can be divided.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleCelled').default}
+                >
+                    <TableSampleCelled />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Collapsing Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+                        If <code>true</code>, applies <code>width: auto</code> to the Table.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleCollapsing').default}
+                >
+                    <TableSampleCollapsing />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Definition Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table may be formatted to emphasize a first column
+                        that defines a row content.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleDefinition').default}
+                >
+                    <TableSampleDefinition />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Fixed Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table column&rsquo;s width can be evenly spaced.
```

Comment:
Should this be `The widths of the table&rsquo;s columns can be evenly spaced.`?
**NAB** is this was the existing wording; I just think it's possibly slightly off.

Reply:
Fixed.

Reply:
Looks good; resolving.